packages/cli/src/tokens/build/transformers.ts

```diff
@@ -12,10 +12,10 @@ export const sizeRem: Transform = {
   type: 'value',
   transitive: true,
   filter: (token) => {
-    const hasWantedType = typeEquals(['dimension', 'fontsize'], token);
-    const hasWantedPath = pathStartsWithOneOf(['spacing', 'sizing', 'border-radius', 'font-size'], token);
+    // const hasWantedType = typeEquals(['dimension'], token);
+    const hasWantedPath = pathStartsWithOneOf(['border-radius'], token);
```

Comment:
Why do we no longer want font-size to be `rem`?

Reply:
Fixed